CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Change Log
 
+## 7.0.0-beta1 - 2019-12-31
```

Comment:
I'd love to start using https://keepachangelog.com/en/1.0.0/ format

Reply:
Same, I've added some headings.